### DISCUSSION TITLE
Double license project

### DISCUSSION THREAD

**Discussion by FerraraDavideG** [community/community#63721] [Repositories]
### Select Topic Area

General

### Body

Hi, my project uses two licenses and i want to publish it on github, i use CERN-OHL-S V2 for hardware and GNU General Purpose License 3.0 for my software. How can i correctly publish my project on the site in this case?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @LAYATORS ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
